buildbot/master.cfg

```diff
@@ -576,6 +577,13 @@ def make_android_package(mode="normal"):
                                descriptionDone="Gradle",
                                haltOnFailure=True))
 
+        master_filename = WithProperties("/srv/http/dl/prs/dolphin-%s.apk", "branchname")
+        url = WithProperties("http://dl.dolphin-emu.org/prs/dolphin-%s.apk", "branchname")
+        
+        f.addStep(FileUpload(slavesrc="Source/Android/app/build/outputs/apk/app-arm_64-release.apk",
```

Comment:
"wip" is for buildbot-try, but you'd need to be authenticated to be allowed to use that.
Only thing left to note is probably the format of the file names for the PR-folder, which looks different for all the other build types (wrt sorting when browsing the page), but +/-0 on keeping it as-is.